examples/first-frame/main.cpp

```diff
@@ -112,42 +112,46 @@ int main(int argc, char *argv[]) {
     command.parseArguments(argc, argv, command_map);
     int result = command.checkArgumentExist(command_map, arg_error);
     if (result != 0) {
-        std::cout << "Argument " << arg_error << " doesn't exist."
-                  << "\n"
-                  << Help_Menu;
+        LOG(ERROR) << "Argument " << arg_error << " doesn't exist! "
+                   << "Please check help menu.";
         return -1;
     }
+
     result = command.helpMenu();
     if (result == 1) {
-        std::cout << Help_Menu;
+        LOG(INFO) << Help_Menu;
```

Comment:
The help menu should not be printed with the LOG tool. Every command line tool I have used so far has printed the help menu directly without any additional text before or after the menu.

Reply:
Solved. Now help menu is printed using std::cout

Reply:
Well, it is still being printed with LOG.

Reply:
Fixed